frontend/src/state/settingsSlice.ts

```diff
@@ -0,0 +1,27 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+export const settingsSlice = createSlice({
+  name: "settings",
+  initialState: {
+    model: localStorage.getItem("model") || "gpt-3.5-turbo-1106",
```

Comment:
We should probably standardize here. We've currently got `gpt-4-0125-preview` as the backend default, though there's some discussion on whether we should change it

Reply:
I changed this to `gpt-4-0125-preview` now for consistency.